staging/src/k8s.io/client-go/discovery/fake/discovery.go

```diff
@@ -29,16 +29,16 @@ import (
 	"k8s.io/client-go/testing"
 )
 
-// FakeDiscovery implements discovery.DiscoveryInterface and sometimes calls testing.Fake.Invoke with an action,
+// Discovery implements discovery.DiscoveryInterface and sometimes calls testing.Fake.Invoke with an action,
 // but doesn't respect the return value if any. There is a way to fake static values like ServerVersion by using the Faked... fields on the struct.
-type FakeDiscovery struct {
+type Discovery struct {
```

Comment:
we cannot change this. It breaks tons of 3rdparty code.

Reply:
OK, thanks.  I will close this PR.